File: bootcamp_c_cpp_dsa-main/bootcamp_c_cpp_dsa-main/C/session2/p8.c
/*
  9 64 7
  36 5 16
  3 4 1
  */
#include<stdio.h>
void main(){

        int rows;
        printf("Enter the number of rows:\n");
        scanf("%d",&rows);

        int num=rows*rows;
        for(int i=1;i<=rows;i++){
                for(int j=1;j<=rows;j++){
                       if(num%2==0){
                       printf("%d ",num*num);
                       num--;
                       }else{
                printf("%d ",num);
                num--;
                       }
                }
	printf("\n");
	}
}

                 

File: java-main/java-main/IO/Nestedfor5/p8.java
/*write a program to print the following pattern
Row =8
$
@ @
& & &
# # # #
$ $ $ $ $
@ @ @ @ @ @
& & & & & & &
# # # # # # # #
*/

import java.util.Scanner;
class Demo{
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        System.out.print("Enter the number of rows: ");
        int row = input.nextInt();
        char ch = '$';

        for(int i = 1; i <= row; i++) {
            for(int j = 1; j <= i; j++) {
                System.out.print(ch++ + " ");
            }
            ch++;

            System.out.println();
        }
    }
}

